config.yaml

```diff
@@ -24,6 +24,7 @@ orgs:
       - abryson-redhat
       - adamgoossens
       - adlerfleurant
+      - adetalhouet
```

Comment:
Can `adetalhouet` be placed above `adlerfleurant` to maintain alphabetic ordering?

Reply:
100%